app/controllers/application_controller.rb

```diff
@@ -62,8 +62,8 @@ def handle_unverified_request
     after_action :remember_theme_id
 
     def remember_theme_id
-      if @theme_id
-        Stylesheet::Watcher.theme_id = @theme_id if defined? Stylesheet::Watcher
+      if @theme_ids.present?
+        Stylesheet::Watcher.theme_id = @theme_ids&.first if defined? Stylesheet::Watcher
```

Comment:
If `@theme_ids.present?` is true, we don't have to use `&.` when calling `.first`